src/main/java/auth/Authentication.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package auth;
+
+import config.*;
+
+import static config.EnvConfigurationFile.dotenv;
+import static io.restassured.RestAssured.given;
+
+public class Authentication {
```

Comment:
create a new package called "salesforce" and move auth, config, endpointurl and entities because all classes are related to the software under test